CourseStream/CourseStream.Data/CourseStreamDBContext.cs

```diff
@@ -4,11 +4,12 @@ namespace CourseStream.Data;
 public class CourseStreamDBContext : DbContext
 {
     public DbSet<CourseStream> CourseStreams { get; set; }
+    public DbSet<Enrollment> EnrolledStudents { get; set; }
```

Comment:
EnrolledStudents also should be renamed to Enrollments

Reply:
Fixed